apps/finance/test/finance.js

```diff
@@ -667,8 +667,8 @@ contract('Finance App', accounts => {
                 })
             })
 
-            it('fails executing disabled payment', async () => {
-                await finance.setPaymentDisabled(1, true)
+            it('fails executing inactive payment', async () => {
+                await finance.setPaymentStatus(1, false)
```

Comment:
We should also add a test to re-enable a payment and see that it can be executed.

Reply:
Good idea, I'll add that!